dozer-ingestion/src/errors.rs

```diff
@@ -218,7 +218,7 @@ pub enum PostgresSchemaError {
     #[error("Column type {0} not supported")]
     ColumnTypeNotSupported(String),
 
-    #[error("CustomTypeNotSupported")]
+    #[error("Custom type is not supported yet. Join our Discord at https://discord.com/invite/3eWXBgJaEQ - we're here to help with your use case!")]
```

Comment:
Shoudl we include the type name ? 

Also if there are multiple types that are not supported. Are we giving them errors for the first type or all of them together ?

Reply:
Right we should include. I believe we quit on first error, @karolisg can you confirm?